src/TLuaInterpreter.cpp

```diff
@@ -10827,6 +10827,57 @@ int TLuaInterpreter::getServerEncodingsList(lua_State* L)
     return 1;
 }
 
+// Determine operating system, moved from a linux/macos/windows only script in
+// "Other.lua" to provide better identification for "other" OSs
```

Comment:
Don't need to know that it was moved in the code comments, won't help anyone 5 years down the road!

Reply:
Okay, will remove it.